asyncpgsa/record.py

```diff
@@ -1,20 +1,29 @@
-
+from sqlalchemy import Column
 
 class Record:
     __slots__ = ('row',)
 
-    def __str__(self):
-        return str(self.row)
-
     def __init__(self, row):
         self.row = row
 
-    def __getattr__(self, item):
+    def __getattr__(self, *args, **kwargs):
         try:
+            item = args[0]
             return self.row[item]
-        except KeyError:
-            raise AttributeError("'Row' object has no attribute '{}'"
-                                 .format(item))
+        except:
+            try:
+                return self.row.__getattribute__(*args, **kwargs)
+            except:
+                pass
+
+        raise AttributeError("'Row' object has no attribute '{}'"
```

Comment:
This is more readable being in the except block. 
Also, your except's should be more explicit about what they are excepting.